geometry_msgs/PointStamped[] ref_object_points
geometry_msgs/PointStamped mystery_object_point
---
int64 mystery_object_num
